gitman/git.py

```diff
@@ -41,7 +50,12 @@ def clone(type, repo, path, *, cache=settings.CACHE, sparse_paths=None, rev=None
     sparse_paths_repo = repo if settings.CACHE_DISABLE else reference
 
     if not settings.CACHE_DISABLE and not os.path.isdir(reference):
-        git("clone", "--mirror", repo, reference)
+        git(
+            *(
+                ["clone", "--mirror", repo, reference]
+                + (user_params if user_params else [])
```

Comment:
Perhaps the first line of this function could be:

```
if user_params is None:
    user_params = []
```

Then this line could be simplified to:

```
git("clone", "--mirror", repo, reference, *user_params)
```